fab/core.py

```diff
@@ -146,6 +149,11 @@ def fab_alpha_div_loss(self, batch_size: int) -> torch.Tensor:
         loss = self.fab_alpha_div_loss_inner(x_ais, log_w_ais)
         return loss
 
+    def target_forward_kl(self, batch_size: int) -> torch.Tensor:
```

Comment:
Use "target_foward_kl" in the config file to use this training setting. Assumes the target has a .sample method